crates/nu-protocol/src/hir.rs

```diff
@@ -909,6 +909,7 @@ pub struct Call {
     pub positional: Option<Vec<SpannedExpression>>,
     pub named: Option<NamedArguments>,
     pub span: Span,
+    pub is_last: bool,
```

Comment:
Externals need this, but perhaps internals may eventually need it. Not a fan of a boolean for this, because it doesn't express its intention well, but we'll keep it for now 🙂